exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -1568,6 +1559,15 @@ func (b *Bitfinex) Subscribe(channelsToSubscribe []stream.ChannelSubscription) e
 	return errs
 }
 
+// ConfigureWS to send checksums and sequence numbers
+func (b *Bitfinex) ConfigureWS() error {
+	checksum := make(map[string]interface{})
+	checksum["event"] = "conf"
+	checksum["flags"] = bitfinexChecksumFlag + bitfinexWsSequenceFlag
+
+	return b.Websocket.Conn.SendJSONMessage(checksum)
```

Comment:
Suggestion: Could even consolidate this even more if you so choose but I will leave that up to you.
```go
	return b.Websocket.Conn.SendJSONMessage(map[string]interface{}{
		"event": "conf",
		"flags": bitfinexChecksumFlag + bitfinexWsSequenceFlag,
	})
```

Reply:
done